index.html

```diff
@@ -343,7 +330,7 @@ <h3>EncryptedMessage</h3>
       </section>
 
       <section id="GraphSignature2012" about="https://w3id.org/security#GraphSignature2012"
-        typeof="rdfs:Class">
+        typeof="sec:Signature">
```

Comment:
@msporny @dlongley @davidlehn @iherman is this kind of annotation actually useful?